packages/localizations/src/de-DE.ts

```diff
@@ -397,7 +398,7 @@ export const deDE: LocalizationResource = {
       actionLink: 'Anmelden',
       actionLink__use_email: 'E-mail nutzen',
       actionLink__use_email_username: 'E-mail oder Benutzernamen nutzen',
-      actionLink__use_passkey: undefined,
+      actionLink__use_passkey: 'Stattdessen Passkey verwenden',
```

Comment:
Let's make it shorter

```suggestion
      actionLink__use_passkey: 'Passkey nutzen',
```